<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_Gestion_RH">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>1021</width>
      <height>611</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabWidgetPage2">
     <attribute name="title">
      <string>Gestion RH</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_basic_rh_tasks">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>951</width>
        <height>541</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Basic RH tasks</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_consulter_empl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>911</width>
          <height>491</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_ajouter_personnel">
         <attribute name="title">
          <string>Ajouter personnel</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>811</width>
            <height>441</height>
           </rect>
          </property>
          <property name="title">
           <string>ajouter personnel</string>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>164</x>
             <y>63</y>
             <width>158</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>first_name [*  Letters only]</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>164</x>
             <y>93</y>
             <width>151</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>last_name [* Letters only]</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>164</x>
             <y>123</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>password</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>160</y>
             <width>134</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>gender  [* 1=m , 0=f ]</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_nom">
           <property name="geometry">
            <rect>
             <x>21</x>
             <y>63</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>ggggggg</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>30</y>
             <width>139</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>email [* valid full Email]</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="geometry">
            <rect>
             <x>21</x>
             <y>123</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_gender">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>160</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_prenom">
           <property name="geometry">
            <rect>
             <x>21</x>
             <y>93</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>rrrrrrrfg</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_email">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>30</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>ggggggg@jghkhg</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_add_employee">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>310</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>add employee</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>160</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>rue [*]</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>200</y>
             <width>169</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>postalcode  [* numbers only]</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_address">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>160</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_postalcode">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>200</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>230</y>
             <width>47</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>role  [*]</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_role">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>230</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_cin">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>68585</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>165</x>
             <y>30</y>
             <width>59</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>id_cin [*] </string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_debitaccount">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>100</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>100</y>
             <width>74</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>debitaccount</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_30">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>130</y>
             <width>77</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>creditaccount</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_31">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>70</y>
             <width>128</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>phone [* numbers only</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_creditaccount">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>130</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_32">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>230</y>
             <width>90</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>score [ 0 to 99]</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_phone">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>70</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>8888888888</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_33">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>190</y>
             <width>83</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>dateNaissance</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_scor">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>230</y>
             <width>137</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit_datenaissance">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>131</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_clear_cases_ajouter">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>310</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>clear cases</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_modifier_personnel">
         <attribute name="title">
          <string>moifier personnel</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>635</width>
            <height>240</height>
           </rect>
          </property>
          <property name="title">
           <string>modifier employer</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_email_2"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_scor_2"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_password_2"/>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>debitaccount</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEdit_debitaccount_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_phone_2"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_role_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>loginstatuts</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_loginstatus_2"/>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>entrer ID</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_id_search_modifier_field">
                <property name="text">
                 <string>5798597961</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>phone</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>email</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>score</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>role</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_postalcode_2"/>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>postalcode</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="lineEdit_creditaccount_2"/>
           </item>
           <item row="4" column="4">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>creditaccount</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="lineEdit_address_2"/>
           </item>
           <item row="5" column="4">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>rue</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="pushButton_clear_cases_ajouter_2">
             <property name="text">
              <string>clear cases</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="pushButton_update_employee">
             <property name="text">
              <string>update content</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_consulter_employee">
         <attribute name="title">
          <string>consulter/supprime employee</string>
         </attribute>
         <widget class="QTableView" name="tableView_afficher_ts">
          <property name="geometry">
           <rect>
            <x>205</x>
            <y>91</y>
            <width>691</width>
            <height>361</height>
           </rect>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>881</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title">
           <string>information about employee database</string>
          </property>
          <widget class="QLabel" name="label_existing_status_to_delete">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>254</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>[+employees stored count =      ]</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>181</width>
            <height>171</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="title">
           <string>supprimer employee</string>
          </property>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>151</width>
             <height>111</height>
            </rect>
           </property>
           <property name="title">
            <string>Entrer ID</string>
           </property>
           <widget class="QLineEdit" name="lineEdit_id_supprimer_search">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>113</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_supprimer_choix">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>121</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>supprimer choix</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Advanced RH options</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
        <property name="text">
         <string>Email diffuser</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
        <property name="text">
         <string>caluculer salaire</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
        <property name="text">
         <string>personnel statistic</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
        </property>
        <property name="text">
         <string>trier personnel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_main_databaseb_status">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>[+db connection status]</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
